packages/react-components/src/components/MessageThread.tsx

```diff
@@ -909,27 +936,44 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
             onRenderAvatar={onRenderAvatar}
             showMessageStatus={showMessageStatus}
             messageStatus={messageProps.message.status}
+            onActionButtonClick={onActionButtonClickMemo}
           />
         );
       }
       return <></>;
     },
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [new Date().toDateString(), isNarrow, messageThreadParticipantCount, onRenderAvatar]
+    [
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+      new Date().toDateString(),
+      isNarrow,
+      messageThreadParticipantCount,
+      onRenderAvatar,
+      onActionButtonClickMemo,
+      props.onRenderFileDownloads,
```

Comment:
❤️ for updating this prop list!

Reply:
Can we also remove the eslint-disable on line 946 now?